src/startup/mapping.jsx

```diff
@@ -0,0 +1,12 @@
+import { routes } from 'startup/routes'
+import { Route } from 'react-router-dom'
+import React from 'react'
```

Comment:
Another small comment here: Let's move the `import React from 'react'` line to be the first import. This is a general rule to follow in all your files. 

Why? Because that import is required only because of the limitations of the current version of `React` and, in essence, it doesn't affect the behavior/look of our component. Therefore, we don't really care to read this line when reading the imports used.